frontend/src/pages/modelServing/screens/metrics/MetricsPageToolbar.tsx

```diff
@@ -0,0 +1,57 @@
+import * as React from 'react';
+import {
+  Button,
+  Select,
+  SelectOption,
+  Text,
+  Toolbar,
+  ToolbarContent,
+  ToolbarItem,
+} from '@patternfly/react-core';
+import { SyncAltIcon } from '@patternfly/react-icons';
+import { isTimeframeTitle } from './utils';
+import { ModelServingMetricsContext } from './ModelServingMetricsContext';
+import { TimeframeTitle } from '../types';
+
+const MetricsPageToolbar: React.FC = () => {
+  const [timeframeOpen, setTimeframeOpen] = React.useState(false);
+  const { currentTimeframe, setCurrentTimeframe, refresh, lastUpdateTime, setLastUpdateTime } =
+    React.useContext(ModelServingMetricsContext);
+  const date = new Date(lastUpdateTime);
+  return (
+    <Toolbar style={{ paddingBottom: 0 }}>
+      <ToolbarContent>
+        <ToolbarItem>
+          <Select
+            isOpen={timeframeOpen}
+            onToggle={(expanded) => setTimeframeOpen(expanded)}
+            onSelect={(e, selection) => {
+              if (isTimeframeTitle(selection)) {
+                setCurrentTimeframe(selection);
+                setTimeframeOpen(false);
+                setLastUpdateTime(Date.now());
+              }
+            }}
+            selections={currentTimeframe}
+          >
+            {Object.values(TimeframeTitle).map((value, index) => (
+              <SelectOption key={index} value={value} />
+            ))}
+          </Select>
+        </ToolbarItem>
+        <ToolbarItem>
+          <Button variant="plain" onClick={refresh}>
+            <SyncAltIcon />
+          </Button>
+        </ToolbarItem>
+        <ToolbarItem alignment={{ default: 'alignRight' }}>
+          <Text component="small">Last update</Text>
+          <br />
+          <Text component="small">{`${date.toLocaleDateString()} ${date.toLocaleTimeString()}`}</Text>
```

Comment:
For the time, I didn't change the format to `just now` or to a relative time for now because I am not sure what is the definition of `just now`. It's 5 mins or 10 mins? Also, I am fetching the metrics automatically and updating it every 5 mins, so if possible the last update time will always be `just now` if I use relative time. What's the best interval to fetch the metrics?

Reply:
Is `just now` in the UX mockups, if so I would leave the date for now and as them to have a more insightful explanation about what just now means.